cairo/tests/ethereum/cancun/test_fork.py

```diff
@@ -487,6 +490,114 @@ def tx_with_sender_in_state(
     return tx, env, chain_id
 
 
+@composite
+def block_strategy(draw, parent_header: Optional[Header] = None):
```

Comment:
Should be replaced by unit test. Fuzzing does not make sense here 
Looking how to import unit test from EELS